github/actions/core.go

```diff
@@ -199,10 +199,10 @@ func (gha *GHA) newAnnotation(T string, a Annotation) {
 	// "::error file={name},line={line},endLine={endLine},title={title}::{message}"
 	// "::error file=app.js,line=1,title=Syntax Error::Missing semicolon"
 	str := fmt.Sprintf("::%s %s::%s", T, strings.Join(attributes, ","), a.Message)
-	gha.println(str)
+	_, _ = gha.println(str)
 }
 
 // println is an internal helper for printing to the expected output io.Writer
-func (gha *GHA) println(i ...interface{}) {
-	fmt.Fprintln(gha.outWriter, i...)
+func (gha *GHA) println(i ...interface{}) (int, error) {
+	return fmt.Fprintln(gha.outWriter, i...)
```

Comment:
Fixes:
```
github/actions/core.go:207:14: Error return value of `fmt.Fprintln` is not checked (errcheck)
        fmt.Fprintln(gha.outWriter, i...)
```

We are propagating the errors upward. Callers are swallowing the errors for now.